github-image-preview.user.js

```diff
@@ -185,7 +186,7 @@
 			let content = "";
 			// not every submodule includes a link; reference examples from
 			// see https://github.com/electron/electron/tree/v1.1.1/vendor
-			const el = $("a", row) || $("div[role='rowheader'] span[title]", row);
+			const el = $("div[role='rowheader'] a, div[role='rowheader'] span[title]", row);
```

Comment:
This is the one line fix for Refined GitHub

Reply:
Thanks!